scripts/NCBI_Assembly/README.md

```diff
@@ -0,0 +1,90 @@
+# Importing NCBI Assembly Data
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+    1. [Download URL](#download-url)
+    2. [Database Overview](#database-overview)
+    3. [Schema Overview](#schema-overview)
+    4. [Notes and Caveats](#notes-and-caveats)
+    5. [License](#license)
+    6. [Dataset Documentation and Relevant Links](#dataset-documentation-and-relevant-links)
```

Comment:
This link is broken (remove?)

Reply:
Added Dataset Documentation and Relevant Links section to the README.md. The link now works.